darwin/options.py

```diff
@@ -80,7 +80,7 @@ def __init__(self):
             default="native",
             help="Frames per second for video split (recommended: 1), use 'native' to use the videos intrinsic fps.",
         )
-        parser_push.add_argument("--frames", action="store_true", help="Annotate a video as independent frames.")
+        parser_push.add_argument("--frames", action="store_true", help="Annotate a video as independent frames")
```

Comment:
Is this change intentional? Seems to just be removing a `.`

Reply:
Fixed!